packages/components-providers/src/I18n/resources.ts

```diff
@@ -0,0 +1,120 @@
+/*
+
+ MIT License
+
+ Copyright (c) 2020 Looker Data Sciences, Inc.
+
+ Permission is hereby granted, free of charge, to any person obtaining a copy
+ of this software and associated documentation files (the "Software"), to deal
+ in the Software without restriction, including without limitation the rights
+ to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ copies of the Software, and to permit persons to whom the Software is
+ furnished to do so, subject to the following conditions:
+
+ The above copyright notice and this permission notice shall be included in all
+ copies or substantial portions of the Software.
+
+ THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ SOFTWARE.
+
+ */
+
+import { Resource } from 'i18next'
+
+export const i18nResources: Resource = {
+  en: {
+    AdvancedInputControls: {
+      Label: 'Clear Field',
```

Comment:
Ah looks like you're mostly calling `t()` with in the value, like `t('Clear Field')` in `AdvancedInputControls.tsx`, but using a different name as the key here in resources ("Label" in this case). For all translations except ones that contain `{{}}`, you can use the value as the key, which is how it's done in the core product, so might as well keep it consistent for sanity's sake. You need to add single quotes if the key contains spaces, so this would be:
```suggestion
      'Clear Field': 'Clear Field',
```
Anytime `t` is called with a key that's not found, an error is thrown and you'll see something like the screenshot below, so to check if everything is properly translated, you can just go to a story or docs page containing that component.
![image](https://user-images.githubusercontent.com/53451193/107575775-22a5d000-6ba5-11eb-8a56-3f739c1e25c5.png)